components/accept-financial-contributions/StartAcceptingFinancialContributionsPage.tsx

```diff
@@ -0,0 +1,530 @@
+import React from 'react';
+import { gql, useQuery } from '@apollo/client';
+import { useRouter } from 'next/router';
+import { defineMessages, FormattedMessage, useIntl } from 'react-intl';
+import styled from 'styled-components';
+
+import { confettiFireworks } from '../../lib/confettis';
+import { API_V2_CONTEXT } from '../../lib/graphql/helpers';
+import { Account, Collective, Host } from '../../lib/graphql/types/v2/graphql';
+import { getCountryDisplayName, isoCountryList } from '../../lib/i18n/countries';
+
+import ApplyToHostModal from '../ApplyToHostModal';
+import HowToUseOpenCollective from '../fiscal-hosting/HowCanAFiscalHostHelpSection';
+import { Box, Flex, Grid } from '../Grid';
+import { getI18nLink } from '../I18nFormatters';
+import Link from '../Link';
+import Loading from '../Loading';
+import StyledButton from '../StyledButton';
+import StyledCard from '../StyledCard';
+import StyledCollectiveCard from '../StyledCollectiveCard';
+import StyledFilters from '../StyledFilters';
+import StyledHr from '../StyledHr';
+import StyledSelect from '../StyledSelect';
+import { H1, P, Span } from '../Text';
+
+export type StartAcceptingFinancialContributionsPageProps = {
+  collective: Collective;
+  onChange: (field: string, value: unknown) => void;
+};
+
+const HostCardContainer = styled(Grid).attrs({
+  justifyItems: 'center',
+  gridGap: '30px',
+  gridTemplateColumns: ['repeat(auto-fill, minmax(250px, 1fr))'],
+  gridAutoRows: ['1fr'],
+})`
+  & > * {
+    padding: 0;
+  }
+`;
+
+const RoundOr = styled.div`
+  width: 64px;
+  height: 64px;
+  border-radius: 50%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: #f1f6ff;
+  color: #1869f5;
+  text-transform: uppercase;
+  font-family: 'Inter';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 18px;
+  margin: 20px 0;
+`;
+
+const Illustration = styled.img`
+  width: 160px;
+  height: 160px;
+  object-fit: cover;
+`;
+
+const ApplyButton = styled(StyledButton)`
+  visibility: hidden;
+`;
+
+const StyledCollectiveCardWrapper = styled(StyledCollectiveCard)`
+  &:hover ${ApplyButton} {
+    visibility: visible;
+  }
+
+  &:hover {
+    border-color: #1869f5;
+  }
+`;
```

Comment:
@hdiniz : Is this button appearing only on hover the final design? The button only on `:hover` might be a problem on screen readers + devices with no pointing devices I believe ?